Lib update (#1)
PopcodeMobile/popnetwork: CHANGELOG.md

## [0.0.13] - Fix response notify

> - Fix: Now when a request is made a notify is send to all listeners.

Reviewer: ```suggestion
- Fix: Now when a request is made a notification is sent to all listeners.
```

## [0.0.14] - Update Lib

> - DIO dependency update.

Reviewer: ```suggestion
- DIO dependency update;
```

> - Adding personality error return in mock;

Reviewer: ```suggestion
- Adding custom error return in mock.
```
